models/wrf/tutorial/README.rst

```diff
@@ -27,7 +27,7 @@ forum for WRF-specific assistance.
 
 If you are new to DART, we recommend that you become familiar with DART
 by working through the :doc:`../../../theory/readme` and then
-understanding the :doc:`DART getting started <../../../README>` documentation.
+understanding the :ref:`DART getting started <Welcome page>` documentation.
```

Comment:
```suggestion
understanding the :ref:`Getting Started <Welcome page>` documentation.
```

Reply:
I think this is to be clear that a user should read the DART getting started documentation (rather than just WRF).

Reply:
Yeah that's a good point